DllImportGenerator/DllImportGenerator/MarshallingAttributeInfo.cs

```diff
@@ -402,6 +427,7 @@ CountInfo CreateCountInfo(AttributeData marshalUsingData, ImmutableHashSet<strin
         MarshallingInfo CreateInfoFromMarshalAs(
             ITypeSymbol type,
             AttributeData attrData,
+            ImmutableHashSet<string> inspectedElements,
```

Comment:
Seems like this should be `ref` now.

Reply:
We want this to be by-value since we don't want to change the value in higher stack frames.